quint/src/cliCommands.ts

```diff
@@ -957,6 +970,65 @@ function isMatchingTest(match: string | undefined, name: string) {
 }
 
 // Derive the verbosity for simulation and verification routines
-function deriveVerbosity(args: { out: string | undefined; outItf: string | undefined; verbosity: number }): number {
-  return !args.out && !args.outItf ? args.verbosity : 0
+function deriveVerbosity(args: { out: string | undefined; verbosity: number }): number {
+  return args.out ? 0 : args.verbosity
+}
+
+const Placeholders = {
```

Comment:
Can we make it fully capitalized? I don't think we have this patter of capitalization for any values in the code (only classes and types), so it's a bit odd.
```suggestion
const PLACEHOLDERS = {
```

Reply:
Will do

Reply:
463d19fe